src/test/groovy/life/qbic/datamodel/dtos/business/ProjectManagerSpec.groovy

```diff
@@ -41,4 +41,14 @@ class ProjectManagerSpec extends Specification{
         assert projectManager.affiliations.isEmpty()
     }
 
+    def "different project managers result in different objects"() {
+        when: "two different project managers are created"
+        ProjectManager projectManager1 = new ProjectManager.Builder("Andrew", "PM", "a.p@d.ef").build()
+        ProjectManager projectManager2 = new ProjectManager.Builder("Betty", "PM", "b.p@d.ef").build()
+
+        then: "the project managers and their hash code are not equal"
+        projectManager1.hashCode() != projectManager2.hashCode()
+        projectManager1 != projectManager2
+    }
+
```

Comment:
Maybe also adding the @equalsandhash. method to the project manager would be nice since you also added the test here